translations/ja.json

```diff
@@ -954,64 +954,64 @@
+	"unsets an individual value in a minikube config file": "minikube 設定ファイルの個々の設定を解除します",
+	"unsupported or missing driver: {{.name}}": "未サポートのドライバーか、ドライバーが見あたりません: {{.name}}",
 	"update config": "設定を更新します",
-	"usage: minikube addons configure ADDON_NAME": "使用方法: minikube addons configure ADDON_NAME",
-	"usage: minikube addons disable ADDON_NAME": "使用方法: minikube addons disable ADDON_NAME",
-	"usage: minikube addons enable ADDON_NAME": "使用方法: minikube addons enable ADDON_NAME",
-	"usage: minikube addons images ADDON_NAME": "",
-	"usage: minikube addons list": "使用方法: minikube addons list",
-	"usage: minikube addons open ADDON_NAME": "使用方法: minikube addons open ADDON_NAME",
-	"usage: minikube config unset PROPERTY_NAME": "使用方法: minikube config unset PROPERTY_NAME",
-	"usage: minikube delete": "使用方法: minikube delete",
-	"usage: minikube profile [MINIKUBE_PROFILE_NAME]": "使用方法: minikube profile [MINIKUBE_PROFILE_NAME]",
-	"using metrics-server addon, heapster is deprecated": "",
-	"version json failure": "JSON でバージョンを表示するのに失敗しました",
-	"version yaml failure": "YAML でバージョンを表示するのに失敗しました",
-	"zsh completion failed": "zsh の補完が失敗しました",
-	"zsh completion.": "",
-	"{{ .name }}: Suggestion: {{ .suggestion}}": "",
-	"{{ .name }}: {{ .rejection }}": "{{ .name }}: {{ .rejection }}",
-	"{{.Driver}} is currently using the {{.StorageDriver}} storage driver, consider switching to overlay2 for better performance": "",
-	"{{.cluster}} IP has been updated to point at {{.ip}}": "{{.cluster}} の IP アドレスは {{.ip}} へと更新されました",
-	"{{.cluster}} IP was already correctly configured for {{.ip}}": "{{.cluster}} の IP アドレスは {{.ip}} としてすでに正常に設定されています",
-	"{{.count}} nodes stopped.": "{{.count}}台のノードが停止しました。",
-	"{{.count}} node{{if gt .count 1}}s{{end}} stopped.": "",
+	"usage: minikube addons configure ADDON_NAME": "使用法: minikube addons configure ADDON_NAME",
+	"usage: minikube addons disable ADDON_NAME": "使用法: minikube addons disable ADDON_NAME",
+	"usage: minikube addons enable ADDON_NAME": "使用法: minikube addons enable ADDON_NAME",
+	"usage: minikube addons images ADDON_NAME": "使用法: minikube addons images ADDON_NAME",
+	"usage: minikube addons list": "使用法: minikube addons list",
+	"usage: minikube addons open ADDON_NAME": "使用法: minikube addons open ADDON_NAME",
+	"usage: minikube config unset PROPERTY_NAME": "使用法: minikube config unset PROPERTY_NAME",
+	"usage: minikube delete": "使用法: minikube delete",
+	"usage: minikube profile [MINIKUBE_PROFILE_NAME]": "使用法: minikube profile [MINIKUBE_PROFILE_NAME]",
+	"using metrics-server addon, heapster is deprecated": "metrics-server アドオンを使用します (heapster は廃止予定です)",
+	"version json failure": "JSON 形式のバージョン表示に失敗しました",
+	"version yaml failure": "YAML 形式のバージョン表示に失敗しました",
+	"zsh completion failed": "zsh のコマンド補完に失敗しました",
+	"zsh completion.": "zsh のコマンド補完です。",
+	"{{ .name }}: Suggestion: {{ .suggestion}}": "{{ .name }}: 提案: {{ .suggestion}}",
+	"{{ .name }}: {{ .rejection }}": "",
```

Comment:
No translations here?
Is this intended?

Reply:
Yes. Looking at other translations, "" looks no translation (i.e. "{{ .name }}: {{ .rejection }}")

Reply:
If no translations (i.e. `"xxxxxxxx": "",` ), minikube use original text "xxxxxxxx" as is, right?